src/encoded/static/components/award.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import _ from 'underscore';
+import moment from 'moment';
 import { Panel, PanelHeading, PanelBody } from '../libs/bootstrap/panel';
 import DataColors from './datacolors';
 import { FetchedData, Param } from './fetched';
 import * as globals from './globals';
 import { ProjectBadge } from './image';
 import { PickerActions } from './search';
 import StatusLabel from './statuslabel';
-
+import { SortTablePanel, SortTable } from './sorttable';
```

Comment:
Tiny request: I’m trying to keep npm_modules sorted by filename, then '../libs/*' sorted by filename, and finally './*' files sorted by filename, so could you move this line above the './statuslabel' line?